markdown/testing/mockito.md

```diff
@@ -1,78 +1,396 @@
 ---
-type: lecture-cg
-title: "Mocking mit Mockito"
+type: lecture-cg title: "Mocking mit Mockito"
 menuTitle: "Mocking"
 author: "Carsten Gips (FH Bielefeld)"
-weight: 4
-readings:
-  - key: "vogellaJUnit"
-  - key: "junit4"
-  - key: "Kleuker2019"
-  - key: "Osherove2014"
-  - key: "Spillner2012"
-  - key: "fernunihagenJunit"
-tldr: |
-  hier kommt eine tolle inline-zusammenfassung!
-  Formatierung _könnte_ auch **gehen**?
-outcomes:
-  - k1: "**wuppie**"
-  - k2: "*foo*"
-  - k3: "fluppie"
-quizzes:
-  - link: "XYZ"
-    name: "Quiz XXX (ILIAS)"
-assignments:
-  - topic: sheet01
-youtube:
-  - link: ""
-    name: "VL "
-  - link: ""
-    name: "Demo "
-  - link: ""
-    name: "Demo "
-fhmedia:
-  - link: ""
-    name: "VL "
----
+weight: 4 readings:
```

Comment:
@jedi101 Irgendwie sieht der neue YAML-Header ziemlich kaputt aus. Kannst Du den alten bitte wieder einsetzen?

Reply:
@cagix - ähm, hab ich da etwas kaputt gemacht? Ich dachte ich hätte darauf geachtet an dem eigentlichen Template nichts zu ändern. Wie setzte ich den alten YAML-Header ein?

Reply:
naja, schau dir doch einfach an, welche änderungen du in diesem pr vorschlägst - dabei wird der ganze header quasi ent-eingerückt (sagt man das so?). 

beheben: gehe einfach auf einen commit, wo das noch nicht so war (vielleicht der start deines branches), kopiere den header aus der datei raus und gehe dann auf den letzten commit und paste den header wieder rein und comitte. da der pr eh gesquasht wird, ist das okay.

Reply:
Should be fixed with: [b439006](https://github.com/PM-Dungeon/PM-Lecture/pull/426/commits/b4390069ae5ad82075532f3fceb4a02cfc44cf19) and the fix for that fix 👀 [2539baf](https://github.com/PM-Dungeon/PM-Lecture/pull/426/commits/2539baf5642b6aea407c12d82b6faf1562f2a1de)

Reply:
Ich frage mich nur über welches Kabel ich da gestolpert bin um in Bereichen der VL rum zu marodieren in denen ich nichts verloren habe...

Reply:
@cagix - Ich hab das gerade mal getestet woran es liegt. Das ist der komische Autoformatierer von IntelliJ der mir da dazwischen grätscht. Den Header muss ich dann wohl ganz am Ende noch einmal fixen. Weil mir der sonst bei jedem Autoformat flöten geht.

// TODO
- [ ] Fix Header!